packages/scss/src/components/inputs/file/_file.scss

```diff
@@ -1,92 +1,246 @@
 .file {
-	border: 2px dashed _theme("commons.divider.color");
-	border-radius: _theme("commons.border.radius");
-	display: block;
-	max-width: 30rem;
-	padding: _theme("spacings.biggest") _theme("spacings.standard");
+	max-width: 30rem; // magic number
+	min-height: 16rem; // magic number
```

Comment:
Not too big in reponsive ? Maybe add a media queries here ?

Reply:
I had to target less than the height of the content, with margins.
And I have not seen a universal solution for the responsive, I tell myself that it will be done on a case-by-case. What would you have expected?

Reply:
Perhaps we could automaticaly switch to the mod-small apparence under specific breakpoint ? Drag and drop files on mobile is tricky to pull-off and won't be used that often, while at the same time the min-height could be disruptive. To be discussed !

Reply:
I can at least offer a `mod-small` mixin.